_employment/job-seekers/employment-support.md

```diff
@@ -32,55 +32,55 @@ If you are out of work, you may be eligible for [unemployment support](http://ca
 
         <li>
           <h3>
-            <a href="/employment/job-seekers/one-on-one/">One-on-One Assistance:</a>
+            <a href="/employment/job-seekers/one-on-one/">One-on-One Assistance</a>
           </h3>
         </li>
 
         <li>
           <h3>
-            <a href="https://www.vets.gov/veterans-employment-center/skills-translator">Skills Translator</a>
+            <a href="/employment/job_seekers/skills-translator">Skills Translator</a>
           </h3>
         </li>  
 
         <li>
           <h3>
-            <a href="/employment/job-seekers/https://www.vets.gov/veterans-employment-center/createresume/">Resume Builder:</a>
+            <a href="/employment/job-seekers/create-resume">Resume Builder</a>
           </h3>
         </li>
 
         <li>
           <h3>
-            <a href="/employment/job-seekers/alternative/">Alternative Ways to Start Your Career </a>
+            <a href="/employment/job-seekers/alternative/">Alternative Ways to Start Your Career</a>
           </h3>
         </li>  
 
         <li>
           <h3>
-            <a href="https://www.vets.gov/veterans-employment-center/for_job_seekers">Search the Jobs Bank:</a>
+            <a href="/employment/job-seekers/search_jobs">Search the Jobs Bank</a>
```

Comment:
@emilyville same here: just verifying that `search_jobs` should use an underscore instead of a hyphen.